indexer/validate_and_format_contacts.cpp

```diff
@@ -406,4 +416,97 @@ bool ValidateLinePage(string const & page)
   return (domain == "line.me" || strings::EndsWith(domain, ".line.me"));
 }
 
+bool isSocialContactTag(string_view tag)
+{
+  return tag == "contact:instagram" || tag == "contact:facebook" || tag == "contact:twitter"
+         || tag == "contact:vk" || tag == "contact:line";
```

Comment:
A faster version of this code is possible, but if we don't use it at all, it would be even better (see my other comment).

Reply:
This code is used from `editor/xml_feature.cpp` to convert short URLs to full URLs for OSM API. I simplified it with `constexpr` as you proposed.